packages/cloudinary/plugin-types.ts

```
@@ -0,0 +1,24 @@
+import type { CorsairPluginContext, PickAuth } from 'corsair/core';
+import { cloudinaryAuthConfig } from './auth-config';
+import { CloudinarySchema } from './schema';
```

Biome (2.5.5)

[error] All these imports are only used as types.

(lint/style/useImportType)